source/org/zfin/framework/presentation/ZfinJSPFunctions.java

```diff
@@ -409,4 +410,11 @@ public static String getAssetPath(String name) {
 
     }
 
+    public static String getSoftwareBranch() {
+        String softwareVersion = VersionService.getSoftwareVersion();
+        if (softwareVersion.startsWith("release-")) {
```

Comment:
Is there a possibility of null value here for softwareVersion?

Reply:
Good point. Added 'unknown' value in case no git info is available.